mlir/utils/jenkins/Dockerfile

```diff
@@ -12,66 +12,79 @@ RUN apt-get clean all
 
 # --------------------- Section 1: ported from llvm-premerge-checks -----------------
 # Keep this section up-to-date with the upstream
-# https://github.com/google/llvm-premerge-checks/blob/master/containers/base-debian/Dockerfile
+# https://github.com/google/llvm-premerge-checks/blob/main/containers/buildbot-linux/Dockerfile
 RUN echo 'install build dependencies'; \
     apt-get update ;\
     apt-get install -y --no-install-recommends \
-        locales openssh-client gnupg ca-certificates  \
-        zip wget git \
+        locales openssh-client gnupg ca-certificates apt-transport-https \
+        zip wget curl git \
         gdb build-essential  \
         ninja-build \
-        ccache \
-        python3 python3-psutil \
-        python3-pip python3-setuptools \
-        swig python3-dev libedit-dev libncurses5-dev libxml2-dev liblzma-dev golang rsync jq;
-
-RUN apt-get update ;\
-    apt-get upgrade -y ;\
-    apt-get install -y \
-        clang-10 lld-10 clang-tidy-10 clang-format-10 \
-        ;\
-    apt-get clean
-
-RUN echo 'configure locale'; \
-    sed --in-place '/en_US.UTF-8/s/^#//' /etc/locale.gen ;\
-    locale-gen ;\
-    echo 'make python 3 default'; \
-    rm -f /usr/bin/python && ln -s /usr/bin/python3 /usr/bin/python; \
-    pip3 install wheel
-
-# Configure locale
+        libelf-dev libffi-dev gcc-multilib libmpfr-dev libpfm4-dev \
+        python3 python3-psutil python3-pip python3-setuptools \
+        lsb-release software-properties-common \
+        swig python3-dev libedit-dev libncurses5-dev libxml2-dev liblzma-dev golang rsync jq \
+        # for libc++ tests that use the timezone database of the chrono header
+        tzdata \
+        # for llvm installation script
+        sudo \
+        # build scripts
+        nodejs ccache \
+        # shell users
+        less vim
+
+# Install cmake 3.23+ from source.
```

Comment:
Why? I'd install the package from kitware

Reply:
Ask google;  this is copied from llvm-premerge-checks's Dockerfile.